docs/content/aws.md

```diff
@@ -0,0 +1,391 @@
+# Running El Carro Operator on Amazon cloud (AWS)
+
+This guide helps you run El Carro Operator in [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/).
+
+If you prefer to use GKE (Google Kubernetes Engine) to deploy the El
+Carro Operator, stop here and refer to the [Quickstart Guide](quickstart.md) or
+[Quickstart Guide for Oracle 18c XE](quickstart-18c-xe.md).
```

Comment:
```suggestion
If you prefer to use GKE (Google Kubernetes Engine) to deploy the El
Carro Operator, stop here and refer to the [Quickstart Guide](quickstart.md).
```
There's an incoming change to the structure of the docs and referencing just quickstart.md here will be enough.

Reply:
Fixed in next commit